pom.xml

```diff
@@ -54,7 +54,7 @@
         <lombok.version>1.18.14</lombok.version>
         <guava.version>30.1.1-jre</guava.version>
         <joda.time.version>2.10.1</joda.time.version>
-        <jackson.bom.version>2.12.1</jackson.bom.version>
+        <jackson.bom.version>2.13.3</jackson.bom.version>
```

Comment:
Current version of jackson is now 2.13.4 - at a minimum this fixes a downstream dependency in some jackson modules that transitively depend on snakeyaml which has a recently published CVE.